modules/experiment/src/main/resources/schemas/neurosciencegraph/experiment/brainimaging/v0.1.0.json

```diff
@@ -0,0 +1,38 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/morphology/brainimaging/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0",
```

Comment:
Update to neurosciencegraph/commons/activity/v0.1.3

Reply:
Done